.env

```diff
@@ -9,3 +9,6 @@ REACT_APP_AVG_BLOCK_TIME=13200
 REACT_APP_BLOCK_POLL_MS=60000
 REACT_APP_COUNTDOWN_POLL_MS=60000
 REACT_APP_DATA_POLL_MS=600000
+
+
+REACT_APP_ETHEREUM_NODE_URI=https://eth-mainnet.alchemyapi.io/v2/L1SIgIOzW9TRnWpcIPU7tx0EZK3NbVu_
```

Comment:
Was this added for dev purposes and forgot to remove? I believe this is an entry in Vercel env vars